dist/index.html

```diff
@@ -5,11 +5,109 @@
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="container-parent">
+      <div class="container-header">
+        <div class="container-logo">
+          <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" id="mainlogo">
+        </div>
+        <div class="container-navSection">
+          <div class="container-nav">
+
+            <div class="menu">
+              <span class="fas fa-home"></span><p>Home</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-desktop"></span><p>Projects</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-user"></span><p>About Me</p>
+            </div>
+            <div class="menu">
+                <span class="fas fa-envelope"></span><p>Contact</p>
+            </div>
+
+          </div>
+          <div class="container-moto">
+            <h2>Custom Designs · Marketing · Branding · And More</h2>
+          </div>
+        </div>
+      </div>
+      <div class="container-body">
+        <div class="content">
+          <div class="content-left">
+            <span class="ownerContact">
```

Comment:
Stay consistent with your class naming conventions. If we're hyphenating class names (which I think is a good idea when using vanilla JS or jQuery), we should follow that pattern everywhere and ditch the camel case here (and elsewhere).